proto/cmp/services/transport/v1alpha/trip_types.proto

```diff
@@ -84,41 +87,41 @@ message TripSegment {
   // flight: "Y", "J", "F".
   // train: "1st", "2nd" or "S", "C", "P"
   // transfer: "S", "P", "VIP", "SS"
-  string service_type_code = 8;
+  string service_type_code = 9;
 
   // Service type description examples:
   // flight: "Economy Class", "Business Class", "First Class"
   // train: "First Class", "Second Class" or "standard", "comfort", and "premium",..
   // transfer: "Shuttle", "Private", "VIP Limosine", "Speedy Shuttle",...
-  string service_type_description = 9;
+  string service_type_description = 10;
 
   // Trip Time
   //
   // Ex: Time(hours=2, minutes=45)
```

Comment:
We should fix the comment as the type has changed to `Duration`, the comment is not valid anymore.

Reply:
Correct